python/tests/test_graph_conversions.py

```diff
@@ -0,0 +1,1011 @@
+from raphtory import Graph
+from raphtory import export
+import pandas as pd
+import json
+
+
+def build_graph():
+    edges_df = pd.read_csv("data/network_traffic_edges.csv")
+    edges_df["timestamp"] = pd.to_datetime(edges_df["timestamp"]).astype(
+        "datetime64[ms, UTC]"
+    )
+
+    vertices_df = pd.read_csv("data/network_traffic_vertices.csv")
+    vertices_df["timestamp"] = pd.to_datetime(vertices_df["timestamp"]).astype(
+        "datetime64[ms, UTC]"
+    )
+
+    return Graph.load_from_pandas(
+        edges_df=edges_df,
+        src="source",
+        dst="destination",
+        time="timestamp",
+        props=["data_size_MB"],
+        layer_in_df="transaction_type",
+        const_props=["is_encrypted"],
+        shared_const_props={"datasource": "data/network_traffic_edges.csv"},
+        vertex_df=vertices_df,
+        vertex_col="server_id",
+        vertex_time_col="timestamp",
+        vertex_props=["OS_version", "primary_function", "uptime_days"],
+        vertex_const_props=["server_name", "hardware_type"],
+        vertex_shared_const_props={"datasource": "data/network_traffic_edges.csv"},
+    )
+
+
+# def test_edge_list_df():
+
+# def test_vertex_list_df():
+
```

Comment:
should actually add these tests

Reply:
These are being testing fully, I just forgot to remove the comments - will do now

Reply:
didn't find them scrolling through, probably just missed them in the large diff